protected/views/patient/view.php

```diff
@@ -56,7 +56,7 @@
 			<div class="row">
 				<div class="large-12 column">
 					<div id="no-practice-address" class="alert-box alert with-icon">
-						Patient has no Practitioner practice address, please correct in PAS before printing Practitioner letter.
+						Patient has no Practitioner practice address, please correct by clicking the edit link in the patient summary before printing Practitioner letter.
```

Comment:
I thought this was something that CERA requested, but I can't find it in the meeting notes.
@Vegemash Do you have any notes about this?